.github/workflows/e2e_nightly.yaml

```diff
@@ -37,3 +37,20 @@ jobs:
     secrets: inherit
     with:
       coverage: false # E2E tests do not generate coverage reports
+
+  notify-on-slack:
+    runs-on: ubuntu-latest
+    needs:
+      - e2e-tests
+      - e2e-tests-unreleased-kong
+      - test-reports
+    steps:
+      - name: Notify on Slack for failures of e2e tests run automatically at night
+        if: failure() && github.event_name == 'schedule'
+        uses: 8398a7/action-slack@v3
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+        with:
+          status: ${{ job.status }}
+          fields: repo,message,commit,author,action,eventName,ref,workflow
+          text: 'E2E tests failed for nightly run, please check why.'
```

Comment:
Can we include a URL to link to the workflow run?

Reply:
Or will it be automatically added by the action 🤔 ?

Reply:
It should produce the below
<img width="495" alt="84587112-64844800-ae57-11ea-8007-7ce83a91dae3" src="https://github.com/Kong/kubernetes-ingress-controller/assets/9593424/0a047b46-e23d-40ab-84e5-be3cce866bd4">
so it should be be added